.github/elixir-test-matrix.json

```diff
@@ -1,29 +1,14 @@
 {
-  "otp_version": [
-    "25.1.2",
-    "24.3.4.6",
-    "23.3.4.18"
-  ],
-  "elixir_version": [
-    "1.14.2",
-    "1.13.4",
-    "1.12.3",
-    "1.11.4"
-  ],
-  "rebar3_version": [
-    "3.20"
-  ],
-  "os": [
-    "ubuntu-20.04"
-  ],
-  "exclude": [
-    {
-      "elixir_version": "1.11.4",
-      "erlang_version": "25.1.2"
-    },
+  "otp_version": ["26.x", "25.x", "24.x"],
```

Comment:
Versions need to be pinned to the furthest extent possible because it messes with the caches becoming broken anytime a new version is released. We don't really care if it's not running the absolute newest patch for everything.